drivers/crypto/Kconfig

```diff
@@ -53,6 +53,7 @@ config CRYPTO_MBEDTLS_SHIM
 	select MBEDTLS
 	select MBEDTLS_ENABLE_HEAP
 	select MBEDTLS_SHA512
+	select MBEDTLS_CIPHER_AES_ENABLED
```

Comment:
There was a recent PR merged that changed all the MBEDTLS_<algo>_ENABLED to just MBEDTLS_<algo>. Not sure the reason but we should confirm the consistency of naming here.

#73267

Reply:
Agreed.

Reply:
yep

Reply:
> There was a recent PR merged that changed all the MBEDTLS_ENABLED to just MBEDTLS

No, this is not what PR https://github.com/zephyrproject-rtos/zephyr/pull/73267 did. It renamed only the hash algorithm Kconfig options that were wrongly named `CONFIG_MBEDTLS_MAC_<hash_algo>_ENABLED` mainly because of the `MAC` part being misleading. The `ENABLED` part was removed as well to align with the name of the Mbed TLS defines (minus the `_C` suffix used in Mbed TLS) ([config-tls-generic.h:276-314](https://github.com/zephyrproject-rtos/zephyr/pull/73267/files#diff-35af03c2e234822e54b1a5ef8918c465ca47f246d1dc732740e125e6a2935732L276-R314)), which is what we are tending towards. Although Mbed TLS also has defines suffixed with `_ENABLED`, so maybe we'll want to drop the `_ENABLED` part from all the Kconfig options for consistency.not sure what we'll want to do with those.

As Flavio pointed out this renaming work can continue in a subsequent PR.

Reply:
it wasn't a blocker for me but just an FYI